flake8_dunder_all/__init__.py

```diff
@@ -162,7 +162,7 @@ def handle_import(self, node: Union[ast.Import, ast.ImportFrom]) -> None:
 			if not node.col_offset and node.end_lineno > self.last_import:  # type: ignore[union-attr]
 				self.last_import = node.end_lineno  # type: ignore[union-attr]
 		else:
-			if not node.col_offset and node.lineno > self.last_import:
+			if node.lineno > self.last_import:
```

Comment:
I removed the offset condition b/c the type check imports are offset - I expected this might break a test or two but it didn't (at least not in my local runs - ci still running so I might have gone the early crow!).

Reply:
I'm not really sure why the check was there. Perhaps I was being overly cautious.